Tests/StreamChatTests/Database/DTOs/MessageDTO_Tests.swift

```diff
@@ -631,7 +631,7 @@ final class MessageDTO_Tests: XCTestCase {
         XCTAssertEqual(messagePayload.isShadowed, loadedMessage?.isShadowed)
         XCTAssertEqual(
             Set(messagePayload.attachmentIDs(cid: channelId)),
-            loadedMessage.flatMap { Set($0.attachments.map(\.attachmentID)) }
+            loadedMessage.flatMap { Set($0.attachments.compactMap(\.attachmentID)) }
```

Comment:
`Cannot convert value of type 'Set<AttachmentId?>' to closure result type 'Set<AttachmentId>?'`